pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/ZKMetadataStore.java

```diff
@@ -352,42 +348,38 @@ private void internalStorePut(OpPut opPut) {
                 CreateMode createMode = getCreateMode(opPut.getOptions());
                 asyncCreateFullPathOptimistic(zkc, opPut.getPath(), opPut.getData(), createMode,
                         (rc, path1, ctx, name) -> {
-                            execute(() -> {
```

Comment:
Use the execute is unnecessary, the executor is only used to execute flush, or add the request to the queue in zk callback.